src/ResourceManager/DataLakeStore/AzureRM.DataLakeStore.psd1

```diff
@@ -152,11 +152,7 @@ PrivateData = @{
 
         # ReleaseNotes of this module
         ReleaseNotes = '
-		* Export-AzureRmDataLakeStoreItem (https://github.com/Azure/azure-powershell/blob/adls-data-plane/src/ResourceManager/DataLakeStore/documentation/upcoming-breaking-changes.md) - Deprecated parameters PerFileThreadCount, ConcurrentFileCount and introduced parameter Concurrency
-		* Import-AzureRMDataLakeStoreItem (https://github.com/Azure/azure-powershell/blob/adls-data-plane/src/ResourceManager/DataLakeStore/documentation/upcoming-breaking-changes.md) -Deprecated parametersPerFileThreadCount, ConcurrentFileCount and introduced parameter Concurrency
-		* Get-AzureRMDataLakeStoreItemContent - Fixed the tail behavior for contents greater than 4MB
-		* Set-AzureRMDataLakeStoreItemExpiry - Introduced new parameter set SetRelativeExpiry for setting relative expiration time
-		* Remove-AzureRmDataLakeStoreItem (https://github.com/Azure/azure-powershell/blob/adls-data-plane/src/ResourceManager/DataLakeStore/documentation/upcoming-breaking-changes.md) - Deprecated parameter Clean.
+		* Add Debug functionality to Datalake store dataplane cmdlets
```

Comment:
@rahuldutta90 you can revert the changes made to the `ReleaseNotes` -- we will update this metadata when we release this module again

Reply:
@cormacpayne Reverted it.